server/server.js

```diff
@@ -0,0 +1,65 @@
+const films = require('../src/data');
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const app = express();
+const path = require('path');
+
+app.use(cors());
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+
+
+app.use(express.static(path.join(__dirname, "/../build")));
+
+
+app.get('/api/films', function (req, res) {
+    res.json(films);
+})
+
+app.get('/api/film/:number', function (req, res) {
+    const film = films.find(function (film) {
+        return film.number === Number(req.params.number)
+    });
+    res.send(film)
+})
+
+//отправка данных
+app.post('/api/film', function (req, res) {
+    const film = {
+        number: req.body.number,
+        films: req.body.name,//body все що приходить від клієнта
+        genre: req.body.genre,
+        releaseDate: req.body.releaseDate,
+        countries: req.body.countries,
+        assessment: req.body.assessment,
+        imdbFilm: req.body.imdbFilm
+    };
+    console.log(film)
+    // films.push(film);
+    res.send(film);
+})
+
+app.put('/api/film/:number', function (req, res) {
+    let film = films.find(function (film) {
+        return film.number === Number(req.params.number)
+    });
```

Comment:
You use this function 2 times, make sense to create a separate function and call in both places